src/main.js

```diff
@@ -19,31 +19,63 @@ userNewBody.addEventListener('input', verifyForm);
 
 ideaGallery.addEventListener('click', function(event) {
   event.preventDefault();
-  var activeStarBtn = document.querySelector('.star-active');
-  var inactiveStarBtn = document.querySelector('.star-inactive');
 
   if (event.target.className === 'delete') {
     deleteFromArray(event);
     deleteFromDOM(event);
   }
 
-  if (event.target.className === 'star-active') {
-    inactiveStarBtn.classList.remove('hide');
-    activeStarBtn.classList.add('hide');
-  }
-
   if (event.target.className === 'star-inactive') {
-    inactiveStarBtn.classList.add('hide');
-    activeStarBtn.classList.remove('hide');
+    toggleStarOnData(event);
+    toggleStarOnDOM(event);
+    console.log(savedIdeas);
+  } else if (event.target.className === 'star-active') {
+    toggleStarOffData(event);
+    toggleStarOffDOM(event);
+    console.log(savedIdeas);
```

Comment:
Had to have these tie together, because it will be an either / or situation. Otherwise the code is confused on my end.